.travis.yml

```diff
@@ -5,9 +5,11 @@ python:
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -qq libdb-dev
+  - mkdir wheelhouse
+  - wget 'http://www.well.ox.ac.uk/~jk/wheelhouse/pysam-0.8.2.1-cp27-none-linux_x86_64.whl' -O 'wheelhouse/pysam-0.8.2.1-cp27-none-linux_x86_64.whl'
```

Comment:
Is there any way to wget wheelhouse without embedding the filename? I can make a tar ball out of it if needs be.